### DISCUSSION TITLE
Link no funciona - Python diferente de otros lenguajes de programación

### DISCUSSION THREAD

**Discussion by Moorea-AI** [rramosp/ai4eng.v1#81] [00 - General]
Hola profe buenas tardes,

El link que hay en la pagina principal del curso que lleva a https://www.houseofbots.com/news-detail/12008-1-some-features-which-will-make-python-different-from-other-programming-languages esta caida.

hay alguna otra página diferente de esta en donde podamos aprender un poco más de Python y hacer el comparativo con otros lenguajes de programación?

Muchas gracias,

**Comment by Felipe-RA** [comment]
Saludos Moorea-AI

Gracias por el aviso, ya mismo lo corregimos.

Puedes hacer uso de estos recursos para aprender de python

https://www.w3schools.com/python/

https://pythonbasics.org/

**Reply by rramosp** [reply]
ya actualizamos los enlaces, gracias Moorea, con estos apuntes vamos mejorando el curso!!
